src/promptflow/tests/sdk_cli_test/unittests/test_exceptions.py

```diff
@@ -41,11 +33,13 @@ def test_error_category_with_user_error(self, pf):
         assert error_type == "RunNotFoundError"
         assert error_target == ErrorTarget.CONTROL_PLANE_SDK
         assert error_message == ""
-        assert (
-            "module=promptflow._sdk._orm.run_info, "
-            'code=raise RunNotFoundError(f"Run name {name!r} cannot be found."), '
-            "lineno="
-        ) in error_detail
+        assert is_matching(
```

Comment:
Double check this in python 3.11, since the error reporting is slightly different between python versions

Reply:
I will remove the line information,  and all other checked information is fixed messages, independent of the python version.
![image](https://github.com/microsoft/promptflow/assets/23182548/fba9c1bf-ff8a-4aac-8068-e03d62cd502b)